### DISCUSSION TITLE
Failed to install OpenBoard

### DISCUSSION THREAD

**Discussion by Luismm2311** [Homebrew/discussions#1572] [Casks]
I haven’t been able to install OpenBoard via Homebrew
When I run `brew install --cask openboard`
I get the following:
```
==> Downloading https://github.com/OpenBoard-org/OpenBoard/releases/download/v1.
==> Downloading from https://github-releases.githubusercontent.com/44120512/529d
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: 203741c86d50a7ea63f36b9d9cc607d843fb7ee17729c46764f155f47e356b8b
  Actual: feb8cc1a9182df28a422602a683923d40f650c815d089fa6e15eff7880edf38e
    File: /Users/****/Library/Caches/Homebrew/downloads/5ac2fca234b38653a7973826e84e2850b9dd1506d58cf05fba6e39d5f1a62a90--OpenBoard-1.6.1.dmg
To retry an incomplete download, remove the file above.
```

So I the I run: `brew cleanup && brew update`
then I tried installing it again, same error so I run `rm Users/****/Library/Caches/Homebrew/downloads/5ac2fca234b38653a7973826e84e2850b9dd1506d58cf05fba6e39d5f1a62a90--OpenBoard-1.6.1.dmg` , tried again and nothing.

What should I do?

**Comment by carlocab** [comment]
https://github.com/Homebrew/homebrew-cask/blob/master/doc/reporting_bugs/checksum_does_not_match_error.md
